frontend/TestRun/TestRunInfo.svelte

```diff
@@ -63,6 +63,10 @@
                     <span class="fw-bold">Test:</span>
                     {(test?.pretty_name || test?.name) ?? "#NO_TEST"}
                 </li>
+                <li>
+                    <span class="fw-bold">Test Method:</span>
+                    {test_run.test_method ?? "#NOT_SET"}
```

Comment:
nitpick

it would be nice if it's not available, just don't show it.

it's probably SCT branch without the change, every introduce something into templates, the `#NOT_SET` is just alarming and doesn't help

Reply:
done